packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -336,6 +341,39 @@ abstract class VpcBase extends Resource implements IVpc {
     };
   }
 
+  /**
+   * Adds a VPN Gateway to this VPC
+   */
+  public enableVpnGateway(options: VpnGatewayProps): void {
```

Comment:
I would expect this method to be called in the constructor of `Vpc`. Why isn't it?

Reply:
Done